### ISSUE TITLE
.NET 7 support

### ISSUE THREAD

**Issue by jasonvasquez** [jburzynski/TypeGen#138]
I'm sure this may be known already, but I wanted to get it logged at least.  After upgrading my project to .NET 7, `dotnet-typegen` unfortunately does not work, and results in a series of errors like this:

```
Could not resolve assembly: System.Runtime, Version=7.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a in any of the searched directories: /home/jvasquez/.nuget/packages/; 
```

**Comment by vyrotek** [comment]
Getting the same thing. Hope .NET 7 support is coming soon. :)

**Comment by jburzynski** [comment]
I just released version 4.0.0, which adds .NET 7 support. It also includes changes from #137

**Comment by vyrotek** [comment]
@jburzynski I just tried to pull the latest NuGet version and I'm getting these errors. I also tried creating a brand new empty .NET 7.0 project but got the same errors.

`Error	NU1202	Package dotnet-typegen 4.0.0 is not compatible with net7.0 (.NETCoreApp,Version=v7.0). Package dotnet-typegen 4.0.0 supports: net7.0 (.NETCoreApp,Version=v7.0) / any	ConsoleApp1	ConsoleApp1.csproj	1	`

`Error		Package 'dotnet-typegen 4.0.0' has a package type 'DotnetTool' that is not supported by project 'ConsoleApp1'.`				

`Error	NU1212	Invalid project-package combination for dotnet-typegen 4.0.0. DotnetToolReference project style can only contain references of the DotnetTool type 	ConsoleApp1	C:\Users\jason\Desktop\ConsoleApp1\ConsoleApp1\ConsoleApp1.csproj	1	`
